runtime/src/builtins.rs

```diff
@@ -163,6 +177,7 @@ fn builtin_feature_transitions() -> Vec<BuiltinFeatureTransition> {
                 "zk_token_proof_program",
                 solana_zk_token_sdk::zk_token_proof_program::id(),
                 solana_zk_token_proof_program::process_instruction,
+                1000, // TODO - find value
```

Comment:
@samkim-crypto were you ever able to figure this out?

Reply:
The curve syscall costs were measured and updated, but not yet for native programs because I was not sure if the same way to measure CU's apply for native built-in programs.

I was just informed that the same way to measure CU's for syscalls also applies for native builtin programs, so I think I should be able to compute these costs today.

Reply:
@taozhu-chicago The worst case CU for the proof program is quite high: `407_121` CU.

Here are the CU's for each [instruction](https://github.com/solana-labs/solana/blob/master/zk-token-sdk/src/zk_token_proof_instruction.rs#L12-L72)
- VerifyCloseAccount: `6_012` CU
- VerifyWithdraw: `112_454` CU
- VerifyWithdrawWithheldTokens: `7_943` CU
- VerifyTransfer: `219_290` CU
- VerifyTransferWithFee: `407_121` CU
- VerifyPubkeyValidity: `2_619` CU

Reply:
@samkim-crypto thanks for putting these numbers together. Agree that the most expensive instruction is high - given compute budget defines `pub const DEFAULT_INSTRUCTION_COMPUTE_UNIT_LIMIT: u32 = 200_000`. I'd suggest to use the average of all 6 instructions `125_906 CU` as default, assuming all 6 instructions have about equal chances of execution. Objections?

Reply:
No objections. I think `VerifyTransferWithFee` will be the least used I believe.